Source/WebKit/NetworkProcess/storage/SQLiteStorageArea.cpp

```diff
@@ -379,6 +378,22 @@ Expected<void, StorageError> SQLiteStorageArea::setItem(IPC::Connection::UniqueI
         return makeUnexpected(StorageError::Database);
     }
 
+    return { };
+}
+
+Expected<void, StorageError> SQLiteStorageArea::setItem(const String& key, const String& value)
+{
+    String oldValue;
```

Comment:
what's the point of `oldValue` here?

Reply:
It's not used here but the other overload of `setItem()` needs to pass in `oldValue` since it dispatches a message with that value. Since I wanted to move much of the logic of setting the item to another function to avoid code duplication, I needed to pass in that parameter here as well.